.github/workflows/scale-test-100-gce.yaml

```diff
@@ -160,14 +161,16 @@ jobs:
 
       - name: Deploy cluster
         id: deploy-cluster
-        uses: cilium/scale-tests-action/create-cluster@238d773bd07754bfd693a6b22c94eddf3a12778d # main
+        uses: thorn3r/scale-tests-action/create-cluster@a9f91acd923775b8b90fa23fb1743ddc8b3efc03  # updates for 5k
         timeout-minutes: 30
         with:
           cluster_name: ${{ steps.vars.outputs.cluster_name }}
-          control_plane_size: n1-standard-8
-          control_plane_count: 1
+          control_plane_size: n2-standard-64
```

Comment:
I just double-checked, and upstream runs  `n2-standard-32`. 
From recent test:
```
gce-scale-cluster-master  us-east1-b  n2-standard-32               10.40.0.3    35.237.108.192  RUNNING
```
https://github.com/kubernetes/test-infra/blob/a4f43f0f14feb4ec582db4c8b4c64ac6ca767b5d/config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml#L153